### PR TITLE
Extend telecabin submode and better focu 

### PR DESCRIPTION
**Pull request by ue71603**
fixes https://github.com/NeTEx-CEN/NeTEx/issues/218

### CODE REVIEW

File: xsd/netex_framework/netex_reusableComponents/netex_submode_version.xsd [hunk 1]
```diff
@@ -408,12 +408,29 @@ Local train adapted for running in mountain railway lines.</xsd:documentation>
...
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="cableCar">
+				<xsd:annotation>
+					<xsd:documentation>A cableCar has few and bog cabins.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="lift">
+				<xsd:annotation>
+					<xsd:documentation>lift is the most general term and does also contain vertical elevators</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_submode_version.xsd:423]:
```suggestion
					<xsd:documentation>Lift is the most general term and does also contain vertical elevators.</xsd:documentation>
```

**Review by trurlurl — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_submode_version.xsd [hunk 2]
```diff
@@ -408,12 +408,29 @@ Local train adapted for running in mountain railway lines.</xsd:documentation>
...
+					<xsd:documentation>Lift is the most general term and does also contain vertical elevators.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
 			<xsd:enumeration value="chairLift"/>
 			<xsd:enumeration value="dragLift"/>
-			<xsd:enumeration value="telecabinLink"/>
+			<xsd:enumeration value="paternoster"/>
+			<xsd:enumeration value="telecabinLink">
+				<xsd:annotation>
+					<xsd:documentation>Horizontal telecabines</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_submode_version.xsd:431]:
```suggestion
					<xsd:documentation>Horizontal telecabines.</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_submode_version.xsd [hunk 3]
```diff
@@ -408,12 +408,29 @@ Local train adapted for running in mountain railway lines.</xsd:documentation>
...
+			</xsd:enumeration>
+			<xsd:enumeration value="lift">
+				<xsd:annotation>
+					<xsd:documentation>Lift is the most general term and does also contain vertical elevators.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
 			<xsd:enumeration value="chairLift"/>
 			<xsd:enumeration value="dragLift"/>
-			<xsd:enumeration value="telecabinLink"/>
+			<xsd:enumeration value="paternoster"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_submode_version.xsd:428]:
```suggestion
			<xsd:enumeration value="paternoster">
				<xsd:annotation>
				 	<xsd:documentation>Paternoster lift is a passenger lift which consists of a chain of open compartments.</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
```

**Review by Aurige — APPROVED** [review]

**Review by nick-knowles — COMMENTED** [review]
Doc Wording could be improved
 
**cableCar**   Typo  " A cableCar has few and **bog** cabins. "

Surely "big cabins?"  .  Better as 
_A Cable car  has one or two large cabins that make alternating journeys up and down_

**lift**: improve phrasing
_Lift is a vertical (or sometimes slanting) elevator with a lift cage, usually in a building. The category excludes ski lifts and other cableways._

**paternoster** - add improve phrasing 
_A paternoster lift is a  vertical passenger lift which consists of a chain of open compartments that  passengers  step into as  they pass._ 

**telecabinLink**: Improve phrasing 
_A telecabine connecting  two points with no or little vertical difference_

**Comment by ue71603** [comment]
Better text in an new PR as this one is merged already... https://github.com/NeTEx-CEN/NeTEx/pull/621

**Comment by nick-knowles** [comment]
UML Model updated
image

**Pull request merged**